tools/build-dev-env.js

```diff
@@ -28,5 +31,12 @@ setEnvVariable('OPERATOR_DATA_DIR', `${process.env.HOME}/.lamassu/operatordata`)
 setEnvVariable('OFAC_SOURCES_NAMES', 'sdn_advanced,cons_advanced')
 setEnvVariable('OFAC_SOURCES_URLS', 'https://www.treasury.gov/ofac/downloads/sanctions/1.0/sdn_advanced.xml,https://www.treasury.gov/ofac/downloads/sanctions/1.0/cons_advanced.xml')
 
+setEnvVariable('BTC_NODE_LOCATION', 'remote')
+setEnvVariable('BTC_WALLET_LOCATION', 'local')
+setEnvVariable('BTC_NODE_HOST', '146.190.228.59')
```

Comment:
Is this a trusted node? Can we really use it for development? And if it's operated by us, should it be shared here publicly?

Reply:
It is a trusted node, operated by us. Your question is indeed correct, so I rebased that line to redact it